apps/web/pages/organizations/[organizationId]/invoices/[invoiceId]/components/invoiceActionButtons.tsx

```diff
@@ -37,29 +36,25 @@ export const InvoiceActionButtons = ({ invoice: invoiceFragment }: InvoiceAction
   const [, updateInvoice] = useMutation(InvoiceUpdateMutationDocument)
 
   const handleSendOrWithdrawInvoice = async (data: InvoiceUpdateInput) => {
-    const action = invoice.sendDate ? InvoiceAction.Withdraw : InvoiceAction.Send
     try {
       await updateInvoice({
         id: invoice.id,
         organizationId: invoice.organization.id,
         data: {
-          sendDate: data.sendDate,
+          sendDate: invoice.sendDate ? null : data.sendDate,
```

Comment:
Wouldnt be better to use nullish coalescing here?
```suggestion
          sendDate: invoice.sendDate ?? data.sendDate,
```